client/src/pages/ArticlePage.tsx

```diff
@@ -26,6 +27,10 @@ const ArticlePage = () => {
         getData("/category.list", setCategories);
         window.scrollTo(0, 0);
         },[])
+    
+    useEffect(() => {
```

Comment:
There are 3 useEffects there. Can you call `postData` in one of them to reduce redundancy? Or there is a reason for them to be in different ones?

Reply:
have merged two of them into one